src/bluesky/bundlers.py

```diff
@@ -427,6 +427,13 @@ def emit_event(readings: Optional[dict[str, Reading]] = None, *args, **kwargs):
                     f"{readable_obj} has async read() method and the callback "
                     "passed to subscribe() was not called with Dict[str, Reading]"
                 )
+            assert (
+                readings is not None
+            ), "The callback passed to subscribe() was not called with Dict[str, Reading]"
+            now = ttime.time()
+            if any(r["timestamp"] - now > timestamp_cutoff_delay for r in readings.values()):  # type: ignore
+                print("WARNING: The timestamps in the readings are too old.")
```

Comment:
Is there a logger we can plug into instead here (I have lost track of the current state of art on that in bluesky 😞 )?